### ISSUE TITLE
sys.stdout.write() is missing

### ISSUE THREAD

**Issue by jminor** [skulpt/skulpt#538]
I would like to use sys.stdout.write() and sys.stdout.flush() but they seem to be missing from Skulpt.

I'm trying to print text one character at a time like this:
```
import sys
import time
def typewriter(text):
	for letter in text:
		sys.stdout.write(letter)
		sys.stdout.flush()
		time.sleep(0.1)
	sys.stdout.write("\n")
typewriter("Hello typewriter :)")
```

I've tried to find a workaround using "print letter," (with a comma at the end) but that introduces extra spaces between each letter. The other workarounds for this issue all rely on sys.stdout or __future__. http://stackoverflow.com/questions/255147/how-do-i-keep-python-print-from-adding-newlines-or-spaces

Any help with this would be greatly appreciated :)

**Comment by waywaaard** [comment]
`sys.stdout` is currently not implemented. Before implementing `stdout.write` and `stdout.flush` we need to implement the `TextIOWrapper` class.

**Comment by albertjan** [comment]
fixed by #543

**Comment by eah13** [comment]
Fixed by #543
